services/app/src/components/team/VoterTeamMember.jsx

```diff
@@ -1,75 +1,64 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { playerWithScorePropType, playerPropType } from '../../prop-types/player';
 import './team.css';
-import playerPropType from '../../prop-types/player';
-import { PlayerStatuses } from '../../constants/playerConstants';
-import Skip from '../../static/svg/Skip.svg';
-import CheckMark from "../../static/svg/Checkmark.svg";
-import SkippedIcon from "../../static/svg/SkippedIcon.svg"
+import { useChessBoardContext } from '../../contexts/ChessBoardContext';
+import VoterRow from './VoterRow';
+import VoterStatusIcons from './VoterStatusIcons';
+import VoterRowActionButtons from './VoterRowActionButtons';
+import { GameState } from '../../constants/gameConstants';
+import VoterScoreIcon from './VoterScoreIcon';
 
 const VoterTeamMember = ({ player, skipMove, index, currentPlayerId }) => {
-  const { name, color, id, status } = player;
-
+  const { gameState } = useChessBoardContext();
+  
   return (
-    <div className="team-list-item gap-m align-c  padding-y-sm padding-x-0" data-testid={`list-${name}-${index}`}>
-      <div
-        className="team-list-item-avatar f-center"
-        style={{
-        backgroundColor: `rgb(${color.background.r}, ${color.background.g}, ${color.background.b})`
-      }}
-      >
-        <div
-          style={{
-              color: `rgb(${color.text.r}, ${color.text.g}, ${color.text.b})`
-            }}
-        >
-          {name[0].toUpperCase()}
-        </div>
-      </div>
-    
-      <div className="team-list-item-name">
-        {name}
-        {' '}
-        { currentPlayerId === id && <span className='font-size-xs'>(you)</span> }
-      </div>
-    
-      { status === PlayerStatuses.FigurePlaced && (
-        <img 
-          src={CheckMark} 
-          className="team-list-item-icon" 
-          alt="player done icon" 
-        /> 
-      )}
-
-      { status === PlayerStatuses.MoveSkipped && (
-        <img 
-          src={SkippedIcon} 
-          className="team-list-item-icon" 
-          alt="player skipped icon" 
-        /> 
-      )}
+    <div className="team-list-item gap-m align-c padding-y-sm padding-x-" data-testid={`list-${player.name}-${index}`}>
+      <VoterRow
+        name={player.name}
+        color={player.color}
+        currentPlayerId={currentPlayerId}
+        id={player.id}
+        addOn={(
+          <>
```

Comment:
this can be made a bit cleaner with
```
  const inProgressAddons = [
    <VoterStatusIcons status={player.status} />
  ];

  if (player.id !== currentPlayerId) {
    inProgressAddons.push(
      <VoterRowActionButtons 
        onSkipMove={() => skipMove(player.id)}
        status={player.status}
      />
    );
  }

  const finishedGameAddons = [
    <VoterScoreIcon 
      score={player.score} 
      status={player.status}
    />
  ];
```

and

```
        addOn={(
          <>
            {gameState === GameState.GAME_IN_PROGRESS && inProgressAddons}
            {gameState === GameState.GAME_FINISHED && finishedGameAddons}
          </>
        )}
```